authors/apps/profiles/tests/test_following.py

```diff
@@ -0,0 +1,95 @@
+from rest_framework import status
+from rest_framework.reverse import reverse
+from authors.apps.authentication.tests.base_test import TestBaseCase
+
+
+class FollowUnfollowTest(TestBaseCase):
+    """
+    User follow/ unfollow test cases
+    """
+
+    def login_user2(self):
+        self.client.post(self.signup_url, self.test_user_2, format='json')
+        """This will login an existing user"""
+        response = self.client.post(
+            self.login_url, self.test_user_2, format='json')
+        token = response.data['token']
+        return token
+
+    def followers_following_url(self):
+        url = reverse("profiles:followers", kwargs={
+            "username": self.test_user_2['user']['username']})
+        return url
+
+    def follow_url(self):
+        url = reverse("profiles:follow", kwargs={
+                      "username": self.test_user_2['user']['username']})
+        return url
+
+    def follow(self):
+        self.login_user2()
+        self.signup_user()
+        token = self.login_user().data['token']
+        response = self.client.post(
+            self.follow_url(),
+            HTTP_AUTHORIZATION='Token ' + token, format='json')
+        return response
+
+    def forbidden_403(self, response):
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def ok_200(self, response):
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def not_found_404(self, response):
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+    def test_get_followers_without_authorization(self):
+        response = self.client.get(
+            self.followers_following_url(), format='json')
+        self.forbidden_403(response)
+
+    def test_get_following_without_auth(self):
+        response = self.client.get(
+            self.followers_following_url(), format='json')
+        self.forbidden_403(response)
+
+    def test_get_following_with_auth(self):
+        token = self.login_user2()
+        response = self.client.get(
+            self.followers_following_url(),
+            HTTP_AUTHORIZATION='Token ' + token, format='json')
+        self.ok_200(response)
+
+    def test_get_followers_with_auth(self):
```

Comment:
This test contains repetition from the test on line 56, you can refactor to make your code DRY